bin/v-restore-user

```diff
@@ -376,6 +381,17 @@ if [ "$web" != 'no' ] && [ -n "$WEB_SYSTEM" ]; then
         rebuild_web_domain_conf
     done
 
+    # if there is at least one domain with LE
+    if $ADD_LETSENCRYPT_CRON_JOB ; then
+        # Adding LE autorenew cronjob
+        if [ -z "$(grep v-update-lets $HESTIA/data/users/admin/cron.conf)" ]; then
+            min=$(generate_password '012345' '2')
+            hour=$(generate_password '1234567' '1')
+            cmd="sudo $BIN/v-update-letsencrypt-ssl"
+            $BIN/v-add-cron-job admin "$min" "$hour" '*' '*' '*' "$cmd" > /dev/null
```

Comment:
There is no need to add this line if we add it to the installers

Similar command In upgrade script will make sure the cronjob runs on existing servers

Reply:
But there are can be existing installations without LE cronjob which will get update and after restore of user there will be no cronjob for LE.
So this is patching existing installations.
Is there some way to add cronjob on update?
But I agree with you that this code should be marked as temporary and removed in future.

Reply:
I can remove it from here, but I don't know where it should be added for upgrade.

Reply:
Create a 1.6.0.sh in /install/upgrade/versions/

And use as a template

https://github.com/hestiacp/hestiacp/blob/main/install/upgrade/versions/1.5.9.sh

👍🏼